package.json

```diff
@@ -10,7 +10,7 @@
     "agent-base": "^6.0.2",
     "asn1.js-rfc2560": "^5.0.0",
     "asn1.js-rfc5280": "^3.0.0",
-    "axios": "^1.6.0",
+    "axios": "^1.6.4",
```

Comment:
Axios 1.6.5 was relesed on Jan 5th - we can use even newer version

Reply:
true! fixed